src/startup.rs

```diff
@@ -10,8 +10,8 @@ use std::net::TcpListener;
 pub async fn run(listener: TcpListener, pool: Pool<Postgres>) -> Result<()> {
     let app = Router::new()
         .route("/", get(|| async { "Hello, World!" }))
-        .route("/subscriber", get(routes::subscriber))
-        .route("/subscriber", post(routes::create_subscriber))
+        .route("/api/subscriber", get(routes::subscriber))
```

Comment:
We should rename this to `subscribers` at some point too... since it does return a list.